week6/Homework6-start/Homework6/View/TaskListView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  TaskListView.swift
+//  
+
+import SwiftUI
+
+struct TaskListView: View {
+  @ObservedObject var taskStore: TaskStore
+    @State var showCompletedTasks: Bool
+    @State private var title = ""
+    
+    var matchingTasks: [Task] {
+        var tasks = taskStore.tasks
+        
+        if !title.isEmpty {
+            tasks = tasks.filter {
+                $0.title.lowercased().contains(title.lowercased())
+            }
+        }
+        
+        return tasks
+    }
+    
+  var body: some View {
+    NavigationStack {
+      ScrollViewReader { scrollProxy in
+          List {
+              ForEach(matchingTasks) { task in
```

Comment:
Nice, some students forget that you have to iterate over the computed property and not the original array.